src/features/profile/pages/Profile.perf.test.tsx

```diff
@@ -0,0 +1,39 @@
+import { rest } from 'msw'
+import React from 'react'
+
+import { UserProfileResponse } from 'api/gen'
+import { AuthWrapper } from 'features/auth/context/AuthContext'
+import { Profile } from 'features/profile/pages/Profile'
+import { beneficiaryUser } from 'fixtures/user'
+import { env } from 'libs/environment'
+import { storage } from 'libs/storage'
+import { reactQueryProviderHOC } from 'tests/reactQueryProviderHOC'
+import { server } from 'tests/server'
+import { measurePerformance, screen } from 'tests/utils'
+
+// We mock server instead of hooks to test the real behavior of the component.
+server.use(
+  rest.get<UserProfileResponse>(env.API_BASE_URL + '/native/v1/me', (_req, res, ctx) =>
+    res(ctx.status(200), ctx.json(beneficiaryUser))
+  )
+)
+
+describe('<Profile />', () => {
+  it('Performance test', async () => {
```

Comment:
Tu peux ajouter "Profile" dans le it ? Normalement tu vas voir apparaitre un test de perf pour le profile. 
Je crois que le rapport récupère le it et non le describe + it (ce qui est problématique)